.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules/
 *yarn-error.log
 *npm-error.log
 *__benchmark.snapshot__.json
+
+.idea/
```

Comment:
You can also ignore these globally by adding it to your `~/.gitignore_global`

Reply:
A good tip. But this also saves other people that are like me ;)